style.css

```diff
@@ -0,0 +1,332 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 12px;
+}
+
+.my-logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.my-menu {
+  position: relative;
+  cursor: pointer;
+  user-select: none;
+}
+
+h1 {
+  width: 327px;
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;

```

Comment:
- [ ] Kindly consider sticking to the original design as much as possible. Your typos(font), color, and spacing(margin, padding) are supposed to match with the Figma design.
`To get a specific element style, Kindly make sure you click the element and you will see the styles applied to it on the right side in Figma` :smile: 

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205618203-64f54e6a-b701-49e5-a4b4-9f3237799da4.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/205618313-07a0c85c-2dfc-4cae-b051-e863787eea12.png)